src/views/MailMergeView/index.tsx

```diff
@@ -15,6 +16,40 @@ export default function MailMergeView({
   exampleDonationItems,
 }: mailMergeProps) {
   const [value, setValue] = React.useState(0);
+  const [templates, setTemplates] = useState<{ [key: string]: any }>({});
+
+  useEffect(() => {
+    const fetchTemplates = async () => {
+      try {
+        const response = await fetch('/api/mailMerge', {
+          method: 'GET',
+        });
```

Comment:
We use the "View" pattern to avoid making server calls from the client, since this code actually executes _after_ the page has rendered. Small but often noticeable performance hit. Possible to bring the retrieval out into `mailMerge/page.tsx` with the other server func calls that retrieve data? That'll also massively simplify the code in this view since there won't be a need for a `useState`